packages/radix-ui-themes/src/components/radio-group.tsx

```diff
@@ -3,54 +3,80 @@
 import * as React from 'react';
 import classNames from 'classnames';
 import * as RadioGroupPrimitive from '@radix-ui/react-radio-group';
-import { radioGroupPropDefs } from './radio-group.props.js';
+import { Text } from './text.js';
+import { radioGroupRootPropDefs } from './radio-group.props.js';
 import { extractProps } from '../helpers/index.js';
 import { marginPropDefs } from '../props/index.js';
 
 import type { ComponentPropsWithoutColor } from '../helpers/index.js';
 import type { MarginProps, GetPropDefTypes } from '../props/index.js';
 
+type RadioGroupRootOwnProps = GetPropDefTypes<typeof radioGroupRootPropDefs>;
+type RadioGroupContextValue = RadioGroupRootOwnProps;
+const RadioGroupContext = React.createContext<RadioGroupContextValue>({});
```

Comment:
If we're going to need context here, we'll need the context scoping, similar to what we do in `CheckboxGroup`